data/json/items/gun/9mm.json

```
@@ -522,11 +523,11 @@
     "looks_like": "hk_mp5",
     "type": "GUN",
     "reload_noise_volume": 10,
-    "name": { "str_sp": "Kel-Tec SUB-2000" },
-    "description": "A uniquely designed pistol caliber carbine with an integral folding stock.  It makes use of 9x19mm Glock magazines.",
+    "name": { "str_sp": "Kel-Tec Sub-2000" },
+    "description": "With its claim to fame being its ease of portability when compared to other conventional rifles and carbines, the Kel-Tec SUB-2000 is a uniquely designed pistol calibre carbine sporting an integral folding mechanism and chambered for customary 9x19mm cartridges, making use of regular Glock compatible magazines.  With its mechanism engaged, the barrel assembly swings rearwards and causes the firearm to fold to half its regular length, packing up into a very compact package with ergonomics unfortunately having to take a backseat to allow for the weapon’s extreme portability.  With a mainly polymer constructed receiver, the carbine is relatively light, incredibly transportable when fully folded, and an all-round inventive firearm.",
     "weight": "1930 g",
     "volume": "1240 ml",
-    "longest_side": "742 mm",
+    "longest_side": "370 mm",
```

```suggestion
    "longest_side": "742 mm",
```
See previous statement on MP 40.